styleguide/source/_patterns/01-molecules/08-lists/14-list-topic-related-articles/_list-topic-related-articles.scss

```diff
@@ -1,25 +1,16 @@
-.ama-theme {
```

Comment:
Are you sure you wanted to remove `.ama-theme`?

Reply:
"list_item-topic-related-article" is a custom class. I don't see a conflict.